bika/lims/jsonapi/read.py

```diff
@@ -42,7 +42,9 @@ def read(context, request):
     for index in indexes:
         if index in request:
             if index == 'UID' and safe_unicode(request[index]) == "":
-                logger.error("Request with Null UID. Catalog Name: %s" % catalog_name)
+                msg = 'Request with no UID (catalog='{0}'). Dismissing UID ' \
+                      'while filtering'.format(catalog_name)
+                logger.warn(msg)
```

Comment:
Although we are using Python 2.7.x, please note that in Python 3, `logger.warn` is deprecated, 
https://docs.python.org/3/library/logging.html#logging.Logger.warning
so please use `logger.warning(msg)` instead.